### DISCUSSION TITLE
Copilot subscription disappeared

### DISCUSSION THREAD

**Discussion by andykcheng** [community/community#36190] [Copilot Conversations] [answered]
My copilot subscription suddenly stopped. Tried log out and in. Never have reply on support ticket over 10 days. Any suggestions? I have transaction ID and also credit card statement. 

Is that how they normally treat people? Very disappointed.

**Comment by 3kh0** [comment]
Seems like there might have been a bug. Try going to the [sign-ups page](https://github.com/github-copilot/signup) and see what you can find. Try uninstalling Copilot and reinstalling it. I do not have Copilot but I was part of the beta.

**Reply by andykcheng** [reply]
Thanks. I have been to the sign up page and it is asking me to pay again. Money is only part of the issue, but the ignorance and lack of support is a bigger problem.

**Comment by iamcjwilliams** [comment, marked as answer]
@ximplarandy Sorry for the trouble here, can you provide more information by selecting (View -> Output; select GitHub Copilot from the pulldown)?

**Comment by andykcheng** [comment]
Hello.  Thank you for taking care of this.  The issue has been solved by a member of staff.  Thank you again for looking into this.

**Comment by Tiago-Rocha** [comment]
I'm facing the same issue. I've seen activation failures and out of date extension (though I'm using the latest available on VSCode extensions market place).


```
[INFO] [default] [2022-10-21T15:09:51.472Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:52.038Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 565 ms
[INFO] [fetch] [2022-10-21T15:09:52.039Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.

[INFO] [default] [2022-10-21T15:09:53.112Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:53.409Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:53.574Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 164 ms
[INFO] [fetch] [2022-10-21T15:09:53.575Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.

[INFO] [default] [2022-10-21T15:09:54.602Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:54.882Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:55.930Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:56.094Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 163 ms
[INFO] [fetch] [2022-10-21T15:09:56.095Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.

[INFO] [default] [2022-10-21T15:09:58.854Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:59.258Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:59.464Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:09:59.629Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 165 ms
[INFO] [fetch] [2022-10-21T15:09:59.631Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.

[INFO] [default] [2022-10-21T15:10:02.646Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-10-21T15:10:02.810Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 164 ms
[INFO] [fetch] [2022-10-21T15:10:02.811Z] Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.
```

**Comment by iamcjwilliams** [comment]
@Tiago-Rocha Have you tried to disable and then re-enable the extension, that would force it to deactivate and reactivate itself?

**Reply by Tiago-Rocha** [reply]
@iamcjwilliams thanks for the reply. Yes I've deactivated, uninstalled several times. When doing that it triggers the activation process which fails with the output above. 

If I go to the to https://marketplace.visualstudio.com/items?itemName=GitHub.copilot the version 1.55.7117 is the latest but on VSCode extensions marketplace the version available is v1.31.6194.

The error is quite clear `Your editor's Copilot extension, version "copilot/1.31.6226", is out of date. Please install a newer version to access this service.` How can I get that latest version through VSCode extensions?

**Reply by iamcjwilliams** [reply]
If you're running VSCode 1.67 or earlier, you must first upgrade VSCode to unlock newer versions of the Copilot extension.

**Comment by bramleffers** [comment]
@iamcjwilliams As of today I'm facing the same issues as @Tiago-Rocha.

I activated my yearly subscription on 2022-09-05 02:38AM PDT. Currently the subsciption is gone only the transaction history is there. (Id = 1QXOBXOW)

Tried the following options;

- [x]  Reinstalling VSCode + CoPilot
- [x] Tried IntellJ Rider
- [x] Rebooting
- [x] Trief other networks
- [x] Checked GitHub Copilot status [page](https://www.githubstatus.com/)
- [x] I tried a different machine with VSCode (Resulted in the exact same error as stated below)

Problem is that I'm unable to obtain a valid token  

OUTPUT of the console: 
`[INFO] [auth] [2022-11-24T21:56:13.287Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T21:56:13.288Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-24T22:05:21.854Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:05:21.856Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."`

Please advice. Kind Regards.

**Reply by adityarizqi** [reply]
yeah i have same issue, cc @iamcjwilliams

**Comment by PabloLION** [comment]
The server is down.. It's not your problem

**Reply by afaizalub** [reply]
oh I see....thanks @PabloLION

**Reply by MrDKGE** [reply]
is there a status page somewhere?
The normal one shows it has no issues.
https://www.githubstatus.com/

**Reply by PabloLION** [reply]
I just `ping copilot-proxy.githubusercontent.com` ...

**Reply by bramleffers** [reply]
Thanks for figuring that out! Indeed the status page shows no error.. We'll wait until it's back online again.

**Comment by dmitrij-borchuk** [comment]
Same issue, output:
```
[INFO] [auth] [2022-11-24T22:42:34.866Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:42:34.872Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Comment by senecamanu** [comment]
<img width="563" alt="image" src="https://user-images.githubusercontent.com/40152234/203872712-0810bbfe-7fad-4713-b18d-54b8dcafa575.png">
it's a server error guys.

**Reply by afaizalub** [reply]
it's the firewall maybe....ahhh, let github gets the problem done
